### DISCUSSION TITLE
How to double the USB-Port of Multiplus II GX device?

### DISCUSSION THREAD

**Discussion by geromd** [Louisvdw/dbus-serialbattery#157] [Q&A] [answered]
Hello,

I am using a Victron Multiplus II GX device with an Daly BMS and a 48V 280Ah EVE cells batterie as ESS system with an EM24 Energymeter
I would like to connect the Daly BMS with an USB-RS485 Victron cable and install the driver to my Multiplus II GX device. Just a question how to connect. The Multiplus II GX has only one USB-Port where the EM 24 Energymeter is already connected. How can I use a second USB-Port for the connection oft the Daly BMS?

Will as common USB-HUB or an USB-Splitter work at the USB-Port of the Multiplus? Are there any recommendations what hardware should be used to double the USB-Port?

Thanky you

Gero

**Comment by Louisvdw** [comment, marked as answer]
You can use any USB-hub to expand your available ports. Anyone will work fine.
